idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -375,11 +387,8 @@ def run_stage1(lazy_run = True):
     fastq_files = execute_command_with_output("ls %s/*.%s" % (FASTQ_DIR, FILE_TYPE)).rstrip().split("\n")
 
     # Identify input files and characteristics
-    if len(fastq_files) <= 1:
-        return # only support paired reads for now
-    else:
-        fastq_file_1 = fastq_files[0]
-        fastq_file_2 = fastq_files[1]
+    if len(fastq_files) not in [1, 2]:
+        return # only support either 1 file or 2 (paired) files
```

Comment:
Returning from the middle of a long function occasionally leads to hard to spot bugs being introduced in future.   Perhaps log a line when it happens.